src/test/java/de/tum/in/www1/artemis/course/CourseTestService.java

```diff
@@ -3329,7 +3329,7 @@ public void testGetCoursesForImport() throws Exception {
         for (Course course : coursesExpected) {
             Optional<CourseForImportDTO> found = courses.stream().filter(c -> Objects.equals(c.id(), course.getId())).findFirst();
             assertThat(found).as("Course is available").isPresent();
-            CourseForImportDTO courseFound = found.orElseThrow();
+            found.orElseThrow();
```

Comment:
This has no effect since the assertion above already checks if there is a result.
```suggestion
```